tools/webpack

```diff
@@ -7,7 +7,7 @@ import subprocess
 
 if False:
     # `NoReturn` wasn't introduced until python 3.5.4
-    from typing import NoReturn
+    from typing import NoReturn  # type: ignore # ->warn-unreachable
```

Comment:
[`if TYPE_CHECKING`](https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles) should work here.